src/Core/ServiceWrapper/CLI/StopWaitOption.cs

```diff
@@ -0,0 +1,41 @@
+﻿using CommandLine;
+using System.Threading;
+using WMI;
+
+namespace winsw.CLI
+{
+    [Verb("stopwait", HelpText = "stop the service and wait until it's actually stopped")]
+    public class StopWaitOption : CliOption
```

Comment:
There is a lot of duplication with the `StopOption`  Maybe one class could inherit another and slightly extend the method

Reply:
When overriding the method doesn't I have to implement whole method in again subclass? Is there any pattern on something I can follow to do that?